PREREQUISITES.md

```diff
@@ -174,8 +174,10 @@ install the Intel SGX driver for both standalone and docker builds.
 You need to install the Intel SGX driver whether you build Avalon standalone
 or using Docker.
 
-First install this package:
+Verify your processor supports Intel SGX with `cpuid | grep SGX:` .
+Also verify Intel SGX is enabled in BIOS.
```

Comment:
How about giving the link to this video https://www.youtube.com/watch?v=bca5NcjoEdc

Reply:
I like John's SGX videos but I do not want to require watching a video to install SGX or Avalon.

Reply:
What I meant is it's not clear how to verify if Intel SGX is enabled in BIOS ? or is this not part of documentation scope ?

Reply:
cpuid may not be installed on all distros. For example, I don't think it's part of the ubuntu docker image Avalon uses. 
`cat /proc/cpuinfo | grep sgx` 
might be more portable.